en/flight_modes_fw/mission.md

```diff
@@ -76,18 +76,20 @@ For more information about mission planning, see:
 
 ## Mission Feasibility Checks
 
-PX4 runs some basic sanity checks to determine if a mission is feasible when it is uploaded, and when the vehicle is first armed.
+PX4 runs some basic sanity checks to determine if a mission is feasible when it is uploaded and before executing a mission.
```

Comment:
FMI Does this prevent switching to mission mode or you switch and the mission is in some kind of paused state?

Reply:
It prevents switching into Mission and will instead switch to Hold.

Reply:
Cool. Captured in https://github.com/PX4/PX4-user_guide/pull/3437